include/seqan3/argument_parser/auxiliary.hpp

```diff
@@ -80,11 +80,11 @@ struct argument_parser_meta_data // holds all meta information
     //!\brief A link to github, your website or a wiki page.
     std::string url;
     //!\brief Brief copyright (and/or license) information.
-    std::string short_copyright;
-    /*!\brief Detailed copyright information that will be displayed
-     *        when the user specifies "--copyright" on the command line.
+    std::string copyright;
```

Comment:
```suggestion
    std::string licence;
```
:grin: ?

Reply:
damn it ... -.-'

Reply:
wait, no, this would be correct for the new copyright/license distinction.
The old short_copyright ist just copyright now.
But I messed up something in the tutorial ^^